app/scripts/modules/core/src/pipeline/config/triggers/triggers.html

```diff
@@ -36,6 +36,27 @@
   <page-section key="parameters" label="Parameters" no-wrapper="true" badge="pipeline.parameterConfig.length">
     <parameters pipeline="pipeline"></parameters>
   </page-section>
+  <page-section key="artifacts" label="Expected Artifacts" badge="pipeline.expectedArtifacts.length" no-wrapper="true">
```

Comment:
Is this only a thing for pub/sub, or are there other triggers that will include artifacts?

Reply:
We'll have other triggers include artifacts. Ideally eventually most/all triggers would be able to do this. We have a few starting use cases with pub/sub -> gae and docker -> k8s v2.